designs/virtualization/4.6/Next-run-configuration/Next-run-configuration.md

```diff
@@ -0,0 +1,65 @@
+---
+parent: Virtualization
+version: 4.6
+---
+
+# Editing
+
+When a user clicks to edit/ update one of the attributes (f.e. Flavor) it will open a modal with an alert notification letting the user know that the VM must be restarted in order to see the changes applied. The alert will persist until they restart the VM.
+In general, users will be able to clear the changes and keep what they had, but at the very first time the modal is opened (since the 'Cancel' button has the same functionality as 'Clear changes') in case the user hasn't saved any changes yet, the 'Clear changes' button will not be shown.
+
+To avoid confusion users will be able to view the current settings in The VM Details page.
+
+![user is editing values](img/EditOriginal.png)
+
+At this point the user can either just save the requested changes and postpone the restart or save and restart the VM.
+
+## Restart modal
+
+If the user chooses to save and restart the VM they will be asked again to ensure that and will get into a modal including an inline notification that restarting may be disruptive to some logged in users and a notification that the new setting will be applied upon restart. The notification will include a confirmation checkbox so they would be aware of the effect that the restart action might cause and a link to the logged in users list at the bottom of the VM Details page.
+
+![user chooses to resart the VM](img/RestartModal.png)
+
+## VM Details page
+
+In the VM Details page they will be notified that the changes they have made will get applied only after the VM will be restarted.
+Next to the ‘Flavor’ section we will add a link that will lead to the Flavor modal where they can view the pending changes.
+Only in cases when users have made changes to the part of the YAML which are not exposed in the ui, we will add a link to ‘View pending changes in YAML editor’.
+
+![VM details page](img/vm-details.png)
+
+In cases users made changes to the part of the YAML which are not exposed in the ui we’ll add the option to ‘View pending changes in the YAML editor.
+
+![Users can view the changes in the YAML](img/ViewYAML-editor.png)
```

Comment:
If we can detect changes were made in the YAML I think we should state that in the text
"View pending changes made in YAML" I think we need to know a bit more about the format of the text we could highlight from the YAML that was changed but it would be nice to be able to provide that here too. Otherwise we are just sending them to a YAML to try and find the change.  @jelkosz WDYT?

Reply:
+1